addons/crm_livechat/models/mail_channel.py

```diff
@@ -40,7 +40,8 @@ def _convert_visitor_to_lead(self, partner, channel_partners, key):
         # partners (no user or portal user) to link to the lead.
         customers = self.env['res.partner']
         for customer in channel_partners.partner_id.filtered('partner_share'):
-            if customer.user_ids and all(user._is_public() for user in customer.user_ids):
+            users = customer.with_context(active_test=False).user_ids
```

Comment:
Shouldn't we move it in the for loop generator to prefetch everything using active_test context ?

Reply:
@tde-banana-odoo 👍